static/js/misc.js

```diff
@@ -51,7 +51,18 @@
 
                     label.appendChild(input);
                     label.appendChild(document.createTextNode(' '));
-                    label.appendChild(document.createTextNode(this.textContent));
+
+                    // Hack to bold the definition since we have to strip rST formatting
+                    const [headText, tailText] = this.textContent.split(':', 2);
+                    if (tailText) {
+                        const bold = document.createElement('b'),
+                              defined = document.createTextNode(`${headText}:`);
+                        bold.appendChild(defined);
```

Comment:
updating the textContent directly would be lighter than creating a new text node
```js                        
const bold = document.createElement('b');
bold.textContent = `${headText}:`;
```